app/models/region.rb

```diff
@@ -21,6 +21,11 @@ class Region < ApplicationRecord
   REGION_TYPES = %w[root organization state district block facility].freeze
   enum region_type: REGION_TYPES.zip(REGION_TYPES).to_h
 
+  # Override the auto-generated root method (via enum) to return the one, single root Region
+  def self.root
+    Region.find_by(region_type: :root)
```

Comment:
We should only ever have one root Region.

Reply:
Makes sense, we don't wanna support multiple region trees since we have orgs to split up different deployments in the same instance.